CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.4.4] - 2021-01-13
+
+### Added
+- fix: open data scraper bugfix (#258)
+- docs: improvements to deployment documentation (#219)
+- fix: Upload Files button disappears for R/W users (#242)
+- feat: install R3.6 and system packages required for dev (#238)
+- fix: file not found error in download-env-config script (#217)
+- test: Add github workflow for e2etest run (#228)
+- feat: modify filter criteria for Open Data (#227)
+- docs: delete dead links (#226)
+- fix: changed RStudio server CSP headers to allow uploads from same-origin (#216)
```

Comment:
We usually don't include PR identifiers in change log. And since some of these features were completed as multiple PRs we should probably include all or just leave it out. I think FHIRs change log also doesn't include PR ids. What do you think?

Reply:
Yeah we don't on FHIR -- lets remove them I was leaving them in for context but on second thought it isn't adding any